tests/test_persistence.py

```diff
@@ -257,136 +257,279 @@ def test_update_open_order(limit_buy_order):
+    # 10 minute limit trade on Binance/Kraken
+    # fee: 0.25 %, 0.3% quote
+    # open_rate: 2.00 quote
+    # amount: = 30.0 crypto
+    # stake_amount
+    #     1x, -1x: 60.0  quote
+    # open_value: (amount * open_rate) ± (amount * open_rate * fee)
+    # 0.25% fee
+    #      1x, 3x: 30 * 2 + 30 * 2 * 0.0025 = 60.15 quote
+    # 0.3% fee
+    #      1x, 3x: 30 * 2 + 30 * 2 * 0.003  = 60.18 quote
     trade = Trade(
-        pair='ETH/BTC',
-        stake_amount=0.001,
-        amount=5,
-        open_rate=0.00001099,
+        pair='ADA/USDT',
+        stake_amount=60.0,
+        amount=30.0,
+        open_rate=2.0,
         fee_open=fee.return_value,
         fee_close=fee.return_value,
         exchange='binance',
     )
     trade.open_order_id = 'open_trade'
-    trade.update(limit_buy_order)  # Buy @ 0.00001099
+    trade.update(limit_buy_order_usdt)  # Buy @ 2.0
 
     # Get the open rate price with the standard fee rate
-    assert trade._calc_open_trade_value() == 0.0010024999999225068
+    assert trade._calc_open_trade_value() == 60.15
     trade.fee_open = 0.003
     # Get the open rate price with a custom fee rate
-    assert trade._calc_open_trade_value() == 0.001002999999922468
+    assert trade._calc_open_trade_value() == 60.18
 
 
 @pytest.mark.usefixtures("init_persistence")
-def test_calc_close_trade_price(limit_buy_order, limit_sell_order, fee):
+def test_calc_close_trade_price(limit_buy_order_usdt, limit_sell_order_usdt, fee):
     trade = Trade(
-        pair='ETH/BTC',
-        stake_amount=0.001,
-        amount=5,
-        open_rate=0.00001099,
+        pair='ADA/USDT',
+        stake_amount=60.0,
+        amount=30.0,
+        open_rate=2.0,
         fee_open=fee.return_value,
         fee_close=fee.return_value,
         exchange='binance',
     )
     trade.open_order_id = 'close_trade'
-    trade.update(limit_buy_order)  # Buy @ 0.00001099
+    trade.update(limit_buy_order_usdt)  # Buy @ 2.0
 
     # Get the close rate price with a custom close rate and a regular fee rate
-    assert trade.calc_close_trade_value(rate=0.00001234) == 0.0011200318470471794
-
+    assert trade.calc_close_trade_value(rate=2.5) == 74.8125
     # Get the close rate price with a custom close rate and a custom fee rate
-    assert trade.calc_close_trade_value(rate=0.00001234, fee=0.003) == 0.0011194704275749754
-
+    assert trade.calc_close_trade_value(rate=2.5, fee=0.003) == 74.775
     # Test when we apply a Sell order, and ask price with a custom fee rate
-    trade.update(limit_sell_order)
-    assert trade.calc_close_trade_value(fee=0.005) == 0.0010619972701635854
+    trade.update(limit_sell_order_usdt)
+    assert trade.calc_close_trade_value(fee=0.005) == 65.67
 
 
 @pytest.mark.usefixtures("init_persistence")
-def test_calc_profit(limit_buy_order, limit_sell_order, fee):
+def test_calc_profit(limit_buy_order_usdt, limit_sell_order_usdt, fee):
+    """
+        10 minute limit trade on Binance/Kraken at 1x, 3x leverage
```

Comment:
I think we'll need to be careful with comments / docstrings in this PR.

With our current approach - this will be merged to develop (so having leverage comments here might be confusing to others as it doesn't really correspond to the content of the function).

Leverage however will be merged to `feat/short` - which will (once complete) be merged to develop.
I expect this process to take several months - so having "leverage" commnets in here now will create a disconnect of several months (maybe more - if this particular test isn't adjusted in `feat/short` to fit the docstring after all.

As such - i'd (for now) delete this Docstring (there wasn't one before ...) - and only add it in a PR that's pointed towards feat/short.

Reply:
There not being one before though meant I had to figure their math out. I took the leverage references out